notifications/views/emailtemplate.py

```diff
@@ -122,6 +122,28 @@ def get_context_data(self, **kwargs):
         context = super(CycleEmailTemplateTest, self).get_context_data(**kwargs)
         context['template'] = get_object_or_404(CycleEmailTemplate,
                                                 id=self.kwargs['pk'])
+        company = Company.objects\
+            .filter(group=context['template'].group)\
+            .order_by('?').first()
+        context['company'] = company
+        context['person'] = company.user.order_by('?').first()
+        context['params'] = context['template'].get_parameters()
+
```

Comment:
Add  comment: #TODO  Create a function that takes param values, body_html and returns the formatted text